bpf/lib/common.h

```diff
@@ -160,7 +160,7 @@ struct endpoint_key {
 /* Value of endpoint map */
 struct endpoint_info {
 	__u32		ifindex;
-	__u16		sec_label;
+	__u16		unused; /* used to be sec_label, no longer used */
```

Comment:
So, where is sec_label stored now?

Reply:
It's not. It was only used for debugging and it had almost no value.